src/main/java/com/needle/FsoFso/product/dto/CartDto.java

```diff
@@ -0,0 +1,56 @@
+package com.needle.FsoFso.product.dto;
+
+public class CartDto {
+
+	long id;
+	long memberId;
+	long productId;
+	int quantity;
```

Comment:
```suggestion
	private long id;
	private long memberId;
	private long productId;
	private int quantity;
```

아래 생성자도 위로 올려주세요!
그리고 빈 생성자 없는데 로직 잘 작동하나요?